boost-common/src/main/java/io/openliberty/boost/utils/SpringBootUtil.java

```diff
@@ -206,4 +246,96 @@ public static Properties getSpringBootServerProperties(String buildDir) throws I
         return featuresToAdd;
     }
 
+    /**
+     * Generate Liberty server configuration files based on the Spring Boot application configuration.
+     * 
+     * @param springBootProjectResources - Path to the Spring Boot application classpath resources
+     * @param libertyServerPath - Path to the Liberty server config directory
+     * @param springBootVersion - Version of Spring Boot for the application
+     * @param springBootStarters - List of Spring Boot starters used by this application
+     * @param logger - BoostLogger for information logging
+     * @throws ParserConfigurationException 
+     * @throws IOException 
+     * @throws TransformerException 
+     */
+    public static void generateLibertyServerConfig(String springBootProjectResources, String libertyServerPath, String springBootVersion, List<String> springBootStarters, BoostLoggerI logger) throws ParserConfigurationException, IOException, TransformerException {
+        
+        logger.info("Generating Liberty server configuration");
+        
+        // Generate Liberty configuration
+        LibertyServerConfigGenerator serverConfig = new LibertyServerConfigGenerator(libertyServerPath);
+    
+        // Find and add appropriate springBoot features
+        List<String> featuresNeededForSpringBootApp = getLibertyFeaturesForSpringBoot(springBootVersion,
+                springBootStarters, logger);
+        
+        serverConfig.addFeatures(featuresNeededForSpringBootApp);
+         // Get Spring Boot server properties
+        Properties springBootServerProps = new Properties();
```

Comment:
This is not needed. The getSpringBootServerProperties method at line 276 creates and returns the Properties object. Just declare the variable type on that line.